packages/frontend-shared/src/gql-components/Auth.vue

```diff
@@ -177,6 +182,12 @@ const loginMutationIsPending = computed(() => {
 })
 
 const handleLoginOrContinue = async () => {
+  if (cloudViewer.value && props.showConnectButtonAfterLogin) {
```

Comment:
/nitpick: We could eliminate the duplicate `cloudViewer.value` check here

```
if (cloudViewer.value) {
  if (props.showConnectButtonAfterLogin) {
    emit('connect-project')
  } else {
    emit('continue', true)
  }
  return
}
```

Reply:
this is style preference for me, I avoid nesting conditionals out of habit if it's not a lot of trouble, but yeah this may be fine

Reply:
updated this and one other place, all goo